main.go

```diff
@@ -255,6 +257,16 @@ func setupListeners(tk timekeeper.TimeKeeper, hostname string, loadedConfig *con
 			return nil, errDupeListener
 		}
 		nameMap[name] = true
+		limitedLogger := &log.RateLimitedLogger{
```

Comment:
don't need to duplicated the logger

Reply:
the log context uses the listener type as an argument. Can I pass some default type to reuse the logger across this loop?